lib/beaker/host_prebuilt_steps.rb

```diff
@@ -17,7 +17,7 @@ module HostPrebuiltSteps
     UNIX_PACKAGES = ['curl', 'ntpdate']
     FREEBSD_PACKAGES = ['curl', 'perl5|perl']
     OPENBSD_PACKAGES = ['curl']
-    ARCHLINUX_PACKAGES = ['curl', 'ntp']
+    ARCHLINUX_PACKAGES = ['curl', 'ntp', 'net-tools', 'openssh']
```

Comment:
Btw, normally net-tools is only installed via beaker-hostgenerator. Beaker itself doesn't need net-tools and I already explained why openssh here doesn't make a lot of sense.